### ISSUE TITLE
Using AXSwift with App Sandbox Turned On Triggers Errors

### ISSUE THREAD

**Issue by DavidMolnar-RS** [tmandry/AXSwift#36]
Hello,

We are getting close to app submission to the App Store.  The problem is that Apple requires the app sandbox to be turned on for all apps submitted to the App Store.  Turning on the App Sandbox causes AXSwift to not work.  It returns UnknownUIElement when trying to read the app under control using Application(forProcessID: ).  Then doing anything else to that top-level UIElement returns an error.

Is there anyway around this?

**Comment by soundflix** [comment]
It doesn't look good:
[https://stackoverflow.com/questions/32116095/how-to-use-accessibility-with-sandboxed-app](url)

**Comment by DavidMolnar-RS** [comment]
Yeah, that's pretty much what I've been seeing.  I'll try going the route of getting that entitlement exception from Apple.  If anyone else has any more insight about this please do comment here.
